setup/schemas.sql

```diff
@@ -0,0 +1,77 @@
+USE library_management_system;
+
+IF NOT EXISTS
+    (
+        SELECT TOP 1 1
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_SCHEMA = 'dbo'
+            AND TABLE_NAME = 'book'
+            AND COLUMN_NAME = 'book_id'
+    )
+BEGIN
+    CREATE TABLE book (
+    book_id INT PRIMARY KEY IDENTITY (1, 1),
+    title NVARCHAR(100) NOT NULL,
+    author NVARCHAR(100) NOT NULL,
+    isbn NVARCHAR(17) NOT NULL,
+    genre NVARCHAR(50) NOT NULL,
+    shelf_location NVARCHAR(100) NOT NULL,
+    status BIT DEFAULT 0 NOT NULL,
+    created_at DATETIME DEFAULT GETDATE() NOT NULL,
+    updated_at DATETIME DEFAULT GETDATE() NOT NULL,
+    CONSTRAINT CHK_BookStatus CHECK (status IN ('available', 'borrowed')),
+    CONSTRAINT CHK_GenreEnum CHECK (genre IN (
+        'Thriller', 'Documentary', 'Drama', 'Horror', 'Romance', 'Mystery', 'Crime', 'Comedy', 'Western', 'Children', 'Action', 'Science Fiction', 'Animation','Fantasy', 'Adventure'
+        )),
+);
+END;
+
+GO
+
+IF NOT EXISTS
+    (
+        SELECT TOP 1 1
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_SCHEMA = 'dbo'
+            AND TABLE_NAME = 'borrower'
+            AND COLUMN_NAME = 'borrower_id'
```

Comment:
you dont need COLUMN_NAME, you only need to check out if the table exist, if you want to add or update later on a particular column you would check then against the column name